scripts/template_upload_rhevm.py

```diff
@@ -177,9 +178,10 @@ def template_from_ova(mgmt, username, password, rhevip, edomain, ovaname, ssh_cl
         logger.info("RHEVM:%r successfully created template from ova file:", provider)
     except Exception:
         logger.exception("RHEVM:%r template_from_ova failed:", provider)
+        raise
 
 
-def import_template(mgmt, edomain, sdomain, cluster, temp_template_name, provider):
+def import_template(mgmt, cfme_data, edomain, sdomain, cluster, temp_template_name, provider):
```

Comment:
honestly, I don't like such way to pass data. it makes everyone to follow the way how data is stored in our yamls.

Reply:
I'd rather update this method to accept properly prepared dict or config object

Reply:
ah, nevermind, I didn't notice that I switched from wrapanapi to our internal scripts. ignore above statements.